data/beaconrestapi/src/test/java/tech/pegasys/teku/beaconrestapi/BeaconRestApiV1Test.java

```diff
@@ -182,7 +182,7 @@ void getRouteExists(final String route, final Class<Handler> type) {
 
   @Test
   void shouldHavePutLogLevel() {
-    verify(app).put(eq(PutLogLevel.ROUTE), any(PutLogLevel.class));
+    verify(app).addHandler(eq(HandlerType.PUT), eq(PutLogLevel.ROUTE), any(PutLogLevel.class));
   }
```

Comment:
push this into the same format the others use, should slot into `postParameters`